public/js/mage/adminhtml/form.js

```diff
@@ -598,4 +678,4 @@ function setPostcodeOptional(zipElement, country) {
 
 if (typeof varienGlobalEvents != 'undefined') {
     varienGlobalEvents.attachEventHandler("address_country_changed", onAddressCountryChanged);
-}
\ No newline at end of file
+}
```

Comment:
```suggestion
}

```

Reply:
Do I not already have a newline in this file? My editor always adds one.

Reply:
I think the same about my phpstorm but sometime it misses, I don't even know why is that important anyway

Reply:
Not sure why it's necessary, but I'm so used to it. This is how GitHub shows when a newline was added to the end of the file:
![image](https://github.com/user-attachments/assets/bc3167ab-4ac6-4c2b-93e2-8051941678d3)

Reply:
checked out the file and can confirm it's missing :-)

<img width="778" alt="Screenshot 2024-11-17 alle 20 02 43" src="https://github.com/user-attachments/assets/dc2dd813-41cf-482a-8193-5c6005000131">

Reply:
Hm, I think you are on the wrong branch, the file should have 681 lines.

Reply:
No problem, I've never had emacs fail to insert the line so I think we're good.